cli/tests/functional/pom.xml

```diff
@@ -71,6 +102,8 @@
                     <systemPropertyVariables>
                         <!-- version should be upgraded to 3.0.0 once released-->
                         <helidon.test.version>2.4.0</helidon.test.version>
+                        <helidon.current.test.version>${project.version}</helidon.current.test.version>
```

Comment:
Is that a build-tools version or a cli version ? If yes this is confusing and we should find a better name.

Reply:
Yes it is confusing, this system property will be merged with `helidon.test.version` once major issues causing test failing are fixed.